attributes/default.rb

```diff
@@ -185,11 +185,13 @@
 # NVIDIA
 default['cluster']['nvidia']['enabled'] = 'no'
 default['cluster']['nvidia']['driver_version'] = '470.141.03'
-default['cluster']['nvidia']['cuda_version'] = '11.4'
+default['cluster']['nvidia']['cuda_version'] = '11.7'
+default['cluster']['nvidia']['cuda_sample_version'] = '11.6'
```

Comment:
Minor: the package is `cuda-samples`. What about using the same name in dictionary key, i.e. `cuda_samples_version` and `cuda_samples_url`?